src/WireMock.Net/Transformers/HandleBarsJsonPath.cs

```diff
@@ -1,14 +1,20 @@
-﻿using System;
-using System.Linq;
-using HandlebarsDotNet;
+﻿using HandlebarsDotNet;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using System;
+using System.Linq;
+using WireMock.Util;
 using WireMock.Validation;
 
 namespace WireMock.Transformers
 {
     internal static class HandleBarsJsonPath
     {
+        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings()
+        {
+            DateParseHandling = DateParseHandling.None
+        };
```

Comment:
All looks good. 

Only thing to consider is whether this could/should be common to both by moving it into `JsonUtils.Parse`?
I can't think of why `JsonMatch` would need to deserialise one way and `JsonPath` another, but there could be a good reason they're done separately? :)

Reply:
Yes. Better to use the JsonUtils.Parse to always use DateParseHandling.None.